filter_test.go

```diff
@@ -121,9 +133,45 @@ var testFilters = []compileTest{
 	},
 	compileTest{
 		filterStr:      `(&(objectclass=inetorgperson)(cn=中文))`,
```

Comment:
in case you want to verify the old bytes were wrong, and the new bytes are right: http://play.golang.org/p/GeNmpuSEs7